sqlglot/dialects/clickhouse.py

```diff
@@ -915,6 +915,22 @@ class Generator(generator.Generator):
             exp.DataType.Type.STRUCT,
         }
 
+        def chr_sql(self, expression: exp.Chr):
+            return self.func("char", expression.this)
+
+        def lag_sql(self, expression: exp.Lag):
+            return self.func(
+                "lagInFrame", expression.this, expression.args["offset"], expression.args["default"]
+            )
+
+        def lead_sql(self, expression: exp.Lead):
+            return self.func(
+                "leadInFrame",
+                expression.this,
+                expression.args["offset"],
+                expression.args["default"],
+            )
```

Comment:
Let's move these to `TRANSFORMS` instead of defining methods.

Reply:
Done 665b453e0f112369f85b04ea7705c0646977e89d